core/src/main/scala/org/apache/spark/resource/ResourceProfileManager.scala

```diff
@@ -67,7 +68,7 @@ private[spark] class ResourceProfileManager(sparkConf: SparkConf,
    */
   private[spark] def isSupported(rp: ResourceProfile): Boolean = {
     if (rp.isInstanceOf[TaskResourceProfile] && !dynamicEnabled) {
-      if ((notRunningUnitTests || testExceptionThrown) && !isStandaloneOrLocalCluster) {
+      if ((notRunningUnitTests || testExceptionThrown) && !supportTaskResourceProfile) {
         throw new SparkException("TaskResourceProfiles are only supported for Standalone " +
```

Comment:
this text should change as it says only standalone mode which isn't true anymore.

Reply:
Done, Thx.